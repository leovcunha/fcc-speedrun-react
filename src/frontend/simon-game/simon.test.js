import React from "react";
import renderer from "react-test-renderer";
import Simon from "../simon-game";

describe("simon", () => {
    it("renders without crashing", () => {
        const component = renderer.create(<Simon/>).toJSON();
        expect(component).toMatchSnapshot();
    });
});
