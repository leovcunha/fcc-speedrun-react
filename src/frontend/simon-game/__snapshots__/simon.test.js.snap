// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`simon renders without crashing 1`] = `
<div
  className="simon jumbotron"
>
  <div
    className="row h1 justify-content-center"
  >
    Simon Game
  </div>
  <hr
    className="p-5"
  />
  <div
    className="gamepad row justify-content-center"
  >
    <table>
      <tbody>
        <tr>
          <td
            className="topleft"
            onClick={[Function]}
            style={
              Object {
                "opacity": 1,
              }
            }
          >
            1
          </td>
          <td
            className="topright"
            onClick={[Function]}
            style={
              Object {
                "opacity": 1,
              }
            }
          >
            2
          </td>
        </tr>
        <tr>
          <td
            className="bottomleft"
            onClick={[Function]}
            style={
              Object {
                "opacity": 1,
              }
            }
          >
            3
          </td>
          <td
            className="bottomright"
            onClick={[Function]}
            style={
              Object {
                "opacity": 1,
              }
            }
          >
            4
          </td>
        </tr>
      </tbody>
    </table>
    <div
      className="central-panel justify-content-center"
    >
      <span
        className="title-panel"
      >
        SIMON
      </span>
      <button
        className="button-panel btn btn-dark"
        onClick={[Function]}
        type="button"
      >
        START
      </button>
      <div
        className="strict-hold d-inline-block"
      >
        <button
          aria-pressed="false"
          autoComplete="off"
          className="strict btn btn-danger"
          data-toggle="button"
          onClick={[Function]}
          type="button"
        />
        <small
          className="text-muted"
        >
          strict
        </small>
      </div>
    </div>
  </div>
</div>
`;
